### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Alex-Welsh**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -107,7 +106,7 @@ or ``ssh ubuntu@<ip>``).
    cd
 
    # Clone Kayobe.
-   git clone https://opendev.org/openstack/kayobe.git -b master
+   git clone https://opendev.org/openstack/kayobe.git -b stable/2023.1
```
Comment (markgoddard) [inline README.rst:109]: We shouldn't change this.

File: README.rst [hunk 2]
```diff
@@ -116,7 +115,7 @@ or ``ssh ubuntu@<ip>``).
    # Clone this Kayobe configuration.
    mkdir -p config/src
    cd config/src/
-   git clone https://github.com/stackhpc/a-universe-from-nothing.git kayobe-config -b master
+   git clone https://github.com/stackhpc/a-universe-from-nothing.git kayobe-config -b stable/2023.1
```
Comment (markgoddard) [inline README.rst:118]: We shouldn't change this.

File: README.rst [hunk 3]
```diff
@@ -513,7 +512,7 @@ OpenStack venv:
     . ~/os-venv/bin/activate
 
     # Install barbicanclient
-    pip install python-barbicanclient -c https://releases.openstack.org/constraints/upper/master
+    pip install python-barbicanclient -c https://releases.openstack.org/constraints/upper/2023.1
```
Comment (markgoddard) [inline README.rst:515]: We shouldn't change this.

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (Alex-Welsh) [reply README.rst]: Reverted

File: README.rst [hunk 4]
```diff
@@ -420,8 +419,8 @@ The new containers can be seen running on the controller node:
 
     $ ssh stack@192.168.33.3 docker ps
     CONTAINER ID   IMAGE                                                                        COMMAND                  CREATED       STATUS                 PORTS     NAMES
-    fad79f29afbc   192.168.33.5:4000/openstack.kolla/opensearch-dashboards:2024.1-rocky-9       "dumb-init --single-…"   6 hours ago   Up 6 hours (healthy)             opensearch_dashboards
-    64df77adc709   192.168.33.5:4000/openstack.kolla/opensearch:2024.1-rocky-9                  "dumb-init --single-…"   6 hours ago   Up 6 hours (healthy)             opensearch
```
[the remark this answers is not in the document]
Comment (Alex-Welsh) [reply README.rst]: Reverted

File: README.rst (hunk 3 above)
Comment (Alex-Welsh) [reply README.rst]: Reverted

**Review by markgoddard — APPROVED** [review]

**Pull request merged**